docs/quality-assurance/test-strategy.md

```diff
@@ -48,10 +48,10 @@ Teams should adhere to this strategy when determining their team approach to tes
 
 ### Cross Browser Testing
 
-* Testing should be carried out on the latest version of Chrome as this is the predominant browser for UKHO applications (this applies to manual and automated testing.)
+* Testing should be carried out on the latest version of Chrome; this applies to manual and automated testing.
```

Comment:
I wonder if we can find some wording here that requires a list of browsers to be explicitly specified by the team for external applications? Currently it reads as if it will be acceptable to just test Chrome for an externally facing application unless the PO asks as an exception. Externally facing applications will should be tested on a range of browsers by default (possibly including mobile browsers).